### PR TITLE
feat: keyboard accessible buttons

### PR DESCRIPTION
**Pull request by mwill8886**
resolves: COIL-1889

Added keyboard accessiblility for the buttons and input on the extension

### CODE REVIEW

**Review by chris-jackson-actionqa — COMMENTED** [review]
I wonder if we can add a test to the testing/e2e-extension suite?  Just some simple keyboard interactions.

File: packages/coil-extension/src/popup/components/TipAmountFeedback/RestrictedMessage.tsx [hunk 1]
```diff
@@ -39,7 +39,12 @@ export const RestrictedMessage = () => {
       return (
         <Typography variant='subtitle1'>
           Max tip amount.{' '}
-          <LinkUnderlined onClick={tabOpener(`${coilDomain}/settings/tipping`)}>
+          <LinkUnderlined
+            tabIndex={0}
```
Comment (sublimator) [inline packages/coil-extension/src/popup/components/TipAmountFeedback/RestrictedMessage.tsx:43]: Could this LinkUnderlined thing be a component? 

eg.  `<CoilLink path='/settings/billing'>`

**Comment by sublimator** [comment]
I built this and tested it, and I don't seem to be able to tab to the main amount and edit it (as I can when I click on it)

**Comment by sublimator** [comment]
Given `escape` already closes the popup, having the first tab-able element on the page be the `X` to close the popup seems a waste?

**Review by sublimator — APPROVED** [review]

**Comment by mwill8886** [comment]
> Given `escape` already closes the popup, having the first tab-able element on the page be the `X` to close the popup seems a waste? 

Good point, I'll see about editing that tomorrow.

File: packages/coil-extension/src/popup/components/TipAmountFeedback/RestrictedMessage.tsx (hunk 1 above)
Comment (mwill8886) [reply packages/coil-extension/src/popup/components/TipAmountFeedback/RestrictedMessage.tsx:43]: Yeah I think so, I'll update.

**Pull request merged**